src/styles/blocs/product_card.css

```
@@ -0,0 +1,99 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+  margin-bottom: 40px;
+
+  display: block;
+}
+
+.card__title {
+  display: block;
+
+  margin-bottom: 4px;
```

```suggestion
  display: block;
  margin-bottom: 4px;
```
You don't need blank line here, it's one group of properties